hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug.robot

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Test ozone Debug CLI
+Library             OperatingSystem
+Resource            ../commonlib.robot
+Test Timeout        2 minute
+
+*** Variables ***
+
+
+*** Test Cases ***
+Create Volume,Bucket and put key
+   Execute             ozone sh volume create o3://om/vol1 --quota 100TB
+   Execute             ozone sh bucket create o3://om/vol1/bucket1
+   Execute             ozone sh key put o3://om/vol1/bucket1/debugKey /opt/hadoop/NOTICE.txt
+
+Test ozone debug
+    ${result} =     Execute             ozone debug chunkinfo o3://om/vol1/bucket1/debugKey | jq -r '.[]'
+                    Should contain      ${result}       files
+    ${result} =     Execute             ozone debug chunkinfo o3://om/vol1/bucket1/debugKey | jq -r '.[].files[0]'
+    ${result3} =    Execute             echo "exists"
+    ${result2} =    Execute             test -f ${result} && echo "exists"
+                    Should Be Equal     ${result2}       ${result3}
```

Comment:
`Execute` checks the return code of the command, so I think this should be enough to verify that the file exists:

```
                    Execute             test -f ${result}
```

Better yet, there is a Robot keyword for the same: [`File Should Exist`](http://robotframework.org/robotframework/3.0.2/libraries/OperatingSystem.html#File%20Should%20Exist)

```suggestion
                    File Should Exist             ${result}
```